src/assets/i18n/hi.json

```diff
@@ -0,0 +1,57 @@
+{
+    "direction": "ltr",
+
+	"LoginDescriptionText": "Letify! निचले स्तर वाले उपकरणों के लिए Spotify क्लाइंट",
```

Comment:
You misspelled the project name, it's Litefy.

Reply:
Hi corrected the project name. 
Thanks